content/app/testing/local/testusers/testData.json

```diff
@@ -0,0 +1,962 @@
+{
+    "$schema": "./testData.schema.json",
```

Comment:
Hopefully this will soon be 
```suggestion
    "$schema": "https://altinncdn.no/schema/json/testUsers/testUsers.schema.v1.json",
```

https://github.com/Altinn/altinn-cdn/pull/118